runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json

```
@@ -169,6 +205,18 @@
       "name": "Leather Chaps",
       "xp": 27
     },
+    {
+      "level": 19,
+      "icon": 5352,
+      "name": "Unfired plant pot",
+      "xp": 17.5
```

This should be 20, with the firing of the pot plant being 17.5: https://oldschool.runescape.wiki/w/Unfired_plant_pot, https://oldschool.runescape.wiki/w/Empty_plant_pot
```suggestion
      "xp": 20
```